src/toil/test/batchSystems/batchSystemTest.py

```diff
@@ -456,6 +458,60 @@ def createBatchSystem(self):
         return KubernetesBatchSystem(config=self.config,
                                      maxCores=numCores, maxMemory=1e9, maxDisk=2001)
 
+@needs_kubernetes_installed
```

Comment:
Is it correct that this method was singled out because this test needs the `kubernetes` module but works without `~/.kube/config, $KUBECONFIG, or current pod`?  Are there Googley configs of any sort it requires instead to authenticate?

Reply:
Yeah, this needs the module but doesn't talk on the network at all.